i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-monitoring-centreon-central.md

```diff
@@ -2,34 +2,276 @@
+
+Le Plugin Pack Centreon Centreon Central apporte un modèle d'hôte :
+* App-Monitoring-Centreon-Central-custom
+
+Il apporte les Modèles de Service suivants :
+
+| Alias              | Modèle de service                             | Description                                                                    | Défaut |
+|:-------------------|:----------------------------------------------|:-------------------------------------------------------------------------------|:-------|
+| Broker-Stats       | App-Monitoring-Centreon-Broker-Stats-Central  | Contrôle les statistiques des processus Centreon Broker                        | X      |
+| proc-broker-rrd    | App-Monitoring-Centreon-Process-broker-rrd    | Contrôle permettant de vérifier le fonctionnement du processus Broker RRD      | X      |
+| proc-broker-sql    | App-Monitoring-Centreon-Process-broker-sql    | Contrôle permettant de vérifier le fonctionnement du processus Broker SQL      | X      |
+| proc-centcore      | App-Monitoring-Centreon-Process-centcore      | Contrôle permettant de vérifier le fonctionnement du processus centcore        |        |
+| proc-centengine    | App-Monitoring-Centreon-Process-centengine    | Contrôle permettant de vérifier le fonctionnement du processus centreon-engine | X      |
+| proc-centreontrapd | App-Monitoring-Centreon-Process-centreontrapd | Contrôle permettant de vérifier le fonctionnement du processus centreontrapd   |        |
+| proc-crond         | App-Monitoring-Centreon-Process-crond         | Contrôle permettant de vérifier le fonctionnement du processus crond           | X      |
+| proc-gorgoned      | App-Monitoring-Centreon-Process-gorgoned      | Contrôle permettant de vérifier le fonctionnement du processus gorgoned        | X      |
+| proc-httpd         | App-Monitoring-Centreon-Process-httpd         | Contrôle permettant de vérifier le fonctionnement du processus Apache          | X      |
+| proc-ntpd          | App-Monitoring-Centreon-Process-ntpd          | Contrôle permettant de vérifier le fonctionnement du processus NTP             | X      |
+| proc-snmptrapd     | App-Monitoring-Centreon-Process-snmptrapd     | Contrôle permettant de vérifier le fonctionnement du processus snmptrapd       |        |
+| proc-sshd          | App-Monitoring-Centreon-Process-sshd          | Contrôle permettant de vérifier le fonctionnement du processus sshd            | X      |
+
+### Métriques & statuts collectés
+
+<Tabs groupId="sync">
+<TabItem value="Broker-Stats" label="Broker-Stats">
+
+| Metric Name                      | Unit     |
+|:---------------------------------|:---------|
+| *endpoint*#queued_events         | events   |
+| *endpoint*#speed_events          | events/s |
+| status                           | string   |
+| *endpoint*#unacknowledged_events | events   |
+
+</TabItem>
+
+<TabItem value="proc-broker-rrd" label="proc-broker-rrd">
+
+| Metric Name | Unit   |
+| :---------- | :----- |
+| Status      | string |
```

Comment:
This is not a status here, by default: 

nb_proc 

and if you add --cpu and --memory flags:

```
'nbproc'=1;;1:1;0; 'mem_total'=19873792B;;;0; 'mem_avg'=19873792.00B;;;0; 'cpu_total'=2.03%;;;0;
```